Router.js

```diff
@@ -98,15 +98,7 @@ export function Link ({ className, href, children }) {
             return;
         }
 
-        // prevent full page reload
-        event.preventDefault();
-
-        // update url
-        window.history.pushState({}, "", href);
-
-        // communicate to Routes that URL has changed
-        const navEvent = new PopStateEvent('popstate');
-        window.dispatchEvent(navEvent);
+        navigate(href);
```

Comment:
`event.preventDefault()` is not included in `navigate`.

```suggestion
        // prevent full page reload
        event.preventDefault();

        navigate(href);
```